jsonparse.ps1

```diff
@@ -0,0 +1,83 @@
+$location=$args[0]
+$repo=$args[1]
+$name=$args[2]
+
+$location
+$repo
+$name
+$repo.Title
+$JSON = (Get-Content $location) -join "`n" | ConvertFrom-Json
+
+$base = "https://github.com/"
```

Comment:
Again with my preference for specific, informative names, I'd call this "`$gitHubBaseUri`".